web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -153,6 +179,25 @@ export function SendForm(props: SendFormProps) {
     <Spinner size={12} customClass="inline-block" />
   );
 
+  const handleRecipientChange = useCallback(
+    async (e: ChangeEvent<HTMLInputElement>) => {
+      const value = e.target.value;
+      setRecipient(value);
+      setError({ recipient: '', amount: error?.amount });
```

Comment:
it's weird to touch the amount here. what about: setError({ recipient: ''})?

or setError((error) => { error, ...recipient: ''});